pkg/apis/config/defaults.go

```diff
@@ -52,6 +58,10 @@ func NewDefaultsConfigFromMap(data map[string]string) (*Defaults, error) {
 		}
 	}
 
+	if nc.RevisionTimeoutSeconds > nc.MaxRevisionTimeoutSeconds {
+		return nil, errors.New("revision-timeout-seconds cannot be greater than max-revision-timeout-seconds")
```

Comment:
It may be helpful here to put the values?

```suggestion
		return nil, fmt.Errorf("revision-timeout-seconds (%d) cannot be greater than max-revision-timeout-seconds (%d)", nc.RevisionTimeoutSeconds, nc.MaxRevisionTimeoutSeconds)
```